sesman/sesman.ini.in

```diff
@@ -37,6 +37,8 @@ RestrictOutboundClipboard=none
 ;   false: an alias of none
 ;   yes: an alias of all
 RestrictInboundClipboard=none
+; Set to 'no' to prevent users from logging in with alternate shells
+AllowedAlternateShells=yes
```

Comment:
By default, we tend to comment-out defaulted parameters these days. On an upgrade, config files are rarely touched, so we need to cater for the parameter not being here. It means you cover the upgrade case naturally during module testing.

Reply:
Okay, I commented it out. I already took care of the parameter being missing, so it should be fine now.

I also tested all three variants (`=yes˚, `=no` and commented-out parameter, which resulted in the shell being allowed, so the same as `=yes`)